hphp/runtime/vm/jit/phys-reg-saver.cpp

```diff
@@ -109,13 +81,9 @@ PhysRegSaver::~PhysRegSaver() {
   }
 }
 
-size_t PhysRegSaver::rspAdjustment() const {
-  return m_adjust;
-}
-
 size_t PhysRegSaver::dwordsPushed() const {
-  assertx((m_adjust % sizeof(int64_t)) == 0);
-  return m_regs.size() + m_adjust / sizeof(int64_t);
+  // Round up the number of regs pushed to an even number
+  return (m_regs.size() + 1) ^ 0x1;
```

Comment:
(I think your intent was probably `(m_regs.size() + 1) & ~0x1`, so I just sort of read it as that... but again, even that's not quite right because of the GP/SIMD size difference.)